docs/UserGuide.md

```diff
@@ -41,138 +31,225 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
-### 2.9. Exiting the program : `exit`
+> Redoes the last data-mutating command inputted since the undone point.<br>
+> If a number is entered, redoes that ammount of previous commands instead.<br>
+> The number **must be a positive integer** 1, 2, 3, ...
+> The number must be less than or equal to the number of commands undone.
 
-Exits the program.<br>
-Format: `exit`
+Examples:
+
+* `add Test`<br>
+  `undo`<br>
+  `redo`
+  Redoes adding Test to the todo list.
+* `add Assignment` <br>
+  `add Project`<br>
+  `undo 2`<br>
+  `redo`<br>
+  Redoes `add Assignment`.
 
 ### 2.10. Saving the data
 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
+todo list data are saved in the hard disk automatically after any command that changes the data.<br>
 There is no need to save manually.
 
-## 3. FAQ
+### 2.11. Viewing previous commands and accessing them : 'history'
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with
-       the file that contains the data of your previous Address Book folder.
+Shows previous commands entered
+Format: 'history'
 
-## 4. Command Summary
+> * previous commands are listed in order from latest command to earlier command
+> * alternatively, pressing on the 'up' and 'down' arrow keys on the keyboard will cycle through the commands previously entered
+
+Examples:
+* 'add Test' <br>
+  'history' <br>
+  Shows 'add Test' in the list.
+* 'add Test' <br>
+  you press on the 'up' arrow key. <br>
+  Shows 'add Test' in your input text field.
 
-* **Add**  `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` <br>
-  e.g. `add James Ho p/22224444 e/jamesho@gmail.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+### 2.12. Add alias for commands: 'add alias'
+
+Adds an alias for a command
+Format: 'addalias COMMAND ALIAS'
+
+> * Once added, alias can be used instead of the command to perform operations.
+
+Example:
+* 'addalias history hs' <br>
+  'hs' <br>
+  Shows 'addalias history hs' in the list.
+
+### 2.13. Delete alias for commands: 'delete alias'
+
+Removes an alias for a command
+Format: 'deletealias ALIAS'
+
+### 2.14. View aliases for commands: 'view alias'
+
+Views aliases in the system
+Format: 'viewalias [COMMAND]'
+
+> * Inputting a value for command will only show all aliases for that command.
+> * Otherwise will show all aliases.
+> * Lists aliases in the format 'COMMAND:ALIAS'.
+
+Example:
+* 'addalias history hs' <br>
+  'viewaliases' <br>
+  Shows 'history:hs' in the list.
+
+### 2.15. Change storage location for data: 'change storage location'
+
+Changes the location for the storage file used in this system
+Format: 'setstoragelocation NEWFILELOCATION'
+
+> * All data will be moved to the new file location.
+> * If the file does not exist, the file will be created.
+> * The old file will be removed.
+
+Example:
+* 'setstoragelocation data/savefile.txt'
```

Comment:
I think both of you have valid points. In this case i think since the argument contains a filename, we can make the command word short since it is understood. So probably follow Louis idea might be better.